src/mc.rs

```diff
@@ -64,6 +64,7 @@ pub enum FilterMode {
   SMOOTH = 1,
   SHARP = 2,
   BILINEAR = 3,
+  SWITCHABLE = 4,
```

Comment:
This matches AV1 spec section [6.8.9 Interpolation filter semantics](https://aomediacodec.github.io/av1-spec/#interpolation-filter-semantics).